editor/import/resource_importer_imagefont.cpp

```diff
@@ -291,10 +291,29 @@ Error ResourceImporterImageFont::import(const String &p_source_file, const Strin
 			WARN_PRINT(vformat("Invalid kerning pairs string: \"%s\"", kp));
 			continue;
 		}
+		String from_tokens;
+		for (int i = 0; i < kp_tokens[0].length(); i++) {
+			if (i <= kp_tokens[0].length() - 6 && kp_tokens[0][i] == '\\' && kp_tokens[0][i + 1] == 'u') {
+				char32_t charcode = kp_tokens[0].substr(i + 2, 4).hex_to_int();
+				from_tokens += charcode;
```

Comment:
@bruvzg This bit of code is missing a part to advance past the hex characters and thus includes `u` and whatever 4 hex characters are included after it. I think it just needs `i += 5` here and in the next loop set to fix it. Do I need a demo project for it?